cellxgene_schema_cli/cellxgene_schema/atac_seq.py

```diff
@@ -475,9 +479,12 @@ def prepare_fragment(
     :param write_algorithm:
     :return:
     """
+    step_size = 10_000_000  # 10 million
     for chromosome in chromosomes:
-        logger.info(f"Processing chromosome: {chromosome}")
-        temp_data = sort_fragment(parquet_file, tempdir, chromosome)
-        write_algorithm(temp_data, bgzip_output_file)
-        Path(temp_data).unlink(missing_ok=True)  # clean up temporary file
```

Comment:
why remove this cleanup? seems like it'd be useful if we're increasing the # of files created

Reply:
that's a good question - i didn't notice it had removed it when i first reviewed this. i'm inclined to keep it unless there's a reason to remove it, @Bento007 ?